src/hooks/useSections.ts

```diff
@@ -15,20 +15,27 @@ export default function useSections(
   advancedParameters?: AdvancedParameters
 ) {
   const zeroStateActiveSections = !query.length && zeroStateSections;
-  const [activeSections, setActiveSections] = useState(
+
+  // Define All Sections
+  const [activeSections, setActiveSections] = useState<UserDefinedSection[]>(
     zeroStateActiveSections ? zeroStateSections : sections
   );
-
+  const sectionsRefs = useRef<RefObject<HTMLLIElement>[]>(activeSections.map(() => createRef()));
```

Comment:
Generate refs that will be attached to sections

Reply:
Do we need this to be a ref?

Reply:
@mocca102 did you decide this needs to be a ref?